checkstyle.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.24,4.25,4.26,4.27
-publicVersions=4.28
+archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18,4.19,4.20,4.21,4.22,4.23,4.24,4.25,4.26,4.27,4.28
+publicVersions=4.29
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+4.29.description=Upgrade to Checkstyle 4.29
+4.29.sqVersions=[7.9,LATEST]
+4.29.date=2020-01-30
+4.29.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.29
+4.29.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.29/checkstyle-sonar-plugin-4.29.jar
+
 4.28.description=Upgrade to Checkstyle 4.28
-4.28.sqVersions=[7.9,LATEST]
+4.28.sqVersions=[7.9,7.9*]
```

Comment:
You probably want to re-set the end of this range to 8.1. If you have users on this version with 8.0 or 8.1 and you end the compatibility range at 7.9, the Marketplace won't be able to notify them that there's a compatible upgrade.

Reply:
thank you. changed :)